ts/features/itwallet/common/components/ItwCredentialClaim.tsx

```diff
@@ -390,6 +392,25 @@ export const ItwCredentialClaim = ({
           );
           return <PlainTextClaimItem label={claim.label} claim={fiscalCode} />;
         }
+        if (NestedObjectClaim.is(decoded)) {
+          const nestedClaims = parseClaims(decoded);
```

Comment:
This prevent some claims to disappear when using the "hide claims" feature. What do you think?
```suggestion
        if (NestedObjectClaim.is(_decoded)) {
          const nestedClaims = parseClaims(_decoded);
```

Reply:
Yes, in the last commit I overrode the value I had previously entered. Thank you for noticing. Addressed in [bd8e14f](https://github.com/pagopa/io-app/pull/7332/commits/bd8e14ff492803b652a3116b5960096c4890f792)